datadreamer/utils/config.py

```diff
@@ -21,7 +21,7 @@ class Config(LuxonisConfig):
         List[float], Field(default=[0.8, 0.1, 0.1], min_length=3, max_length=3)
     ] = [0.8, 0.1, 0.1]
     # Prompt generation arguments
-    prompt_generator: Literal["simple", "lm", "tiny"] = "simple"
+    prompt_generator: Literal["simple", "lm", "tiny", "qwen2"] = "qwen2"
```

Comment:
I thought we had "tiny" as the default prompt generator, but it was actually "simple". Do you think we should change it to "qwen2", considering there might be hardware-constrained users?

Reply:
Yeah, I was surprised as well.

IMO, even hardware-constrained users have to run one of the image generator models, which are already quite demanding, so sure, it will prolong the dataset creation running time, but because of the image generation the whole proces already takes relatively some time. In case, teh users wouldn't like this, they still have the option to use the simple prompt generator, they would just have to specify to use the simple prompt generator. Additionally, by setting Qwen2.5 as the default prompt generator, we should end up with more diverse and creative prompts which could potentially result in more diverse and hopefully better-quality generated images.

Reply:
Ok, good points, I agree!